scripts/source-maintenance.sh

```diff
@@ -232,10 +232,8 @@ for FILENAME in `git ls-files`; do
 	chmod 755 ${FILENAME}
 	;;
 
-    Makefile.am)
-
-    	perl -p -e 's/@([A-Z0-9_]+)@/\$($1)/g' <${FILENAME} >${FILENAME}.styled
-	mv ${FILENAME}.styled ${FILENAME}
+    *.am)
+		run_ applyPlugin scripts/sort-makefile-am-sources.pl ${FILENAME}
```

Comment:
We have discussed this safety feature at length when applyPlugin was being added. Let's not accidentally drop it now:

```suggestion
		run_ applyPlugin scripts/sort-makefile-am-sources.pl ${FILENAME} || return
```

Reply:
ok